index.js

```diff
@@ -233,18 +245,22 @@ client.on('messageReactionAdd', async (reaction, user) => {
         }
     }
 
-    if (reaction.message.id === PRONOUN_REACTION_POST_ID) {
+  if (EMOJI_ROLES) {
+    for (const [messageId, emojiToRoleMap] of emojiToRoleMaps) {
```

Comment:
`emojiToRoleMaps` is a `Map` so you can retrieve `emojiToRoleMap` directly.

e.g.:

```js
const emojiToRoleMap = emojiToRoleMaps.get(reaction.message.id);
```